canonical-web-and-design/ubuntu.com
# Integrate alembic

## Done

- Integrate the work around alembic that has been done on assets.ubuntu.com on ubuntu.com

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Make sure site still works
